doc/docs.md

```diff
@@ -1993,15 +1993,18 @@ mut p := Point{
 	x: 10
 	y: 20
 }
+
+// You can omit fied name.
 p = Point{
-	x: 30
-	y: 4
+	30
+	4
 }
+
 assert p.y == 4
-//
+
 // array: first element defines type of array
-points := [Point{10, 20}, Point{20, 30}, Point{40, 50}]
-println(points) // [Point{x: 10, y: 20}, Point{x: 20, y: 30}, Point{x: 40,y: 50}]
+points := [Point{10, 20}, Point{20 30}] // You can omint `,` also like 2nd case.
```

Comment:
```suggestion
points := [Point{10, 20}, Point{20 30}] // You can omit `,` as in the 2nd case.
```

Reply:
It compiles, but `fmt` inserts a comma.
Is it a bug?

Reply:
It's more explicit to have the comma, but if the compiler accepts it, `v fmt` probably shouldn't be changing it.  I don't know if this qualifies as a compiler bug or a `v fmt` bug.

Reply:
> It compiles, but `fmt` inserts a comma. Is it a bug?

no, it is not a bug, please restore these lines as they were

Reply:
So which is correct? checker? fmt?

Reply:
both are correct, and if you do not comment the lines, they will be checked that:
a) the code in them compiles
b) the code in them is invariant under vfmt

Reply:
Are they both correct? I don't understand.
Since you are giving preference to fmt, your answer is practically equivalent to saying that fmt is correct.